### DISCUSSION TITLE
router upgrade tov0.45.141-beta

### DISCUSSION THREAD

**Discussion by laosuye** [vernesong/OpenClash#3475] [Q&A]
router upgrade tov0.45.141-beta，OpenClash paralysis lost after reboot

**Comment by syjdzh** [comment]
我更新了这个客户端后，也打不开openclash了，不知道什么原因

**Comment by laosuye** [comment]
已经正常升级成功，谢谢！

syjdzh ***@***.***> 于2023年8月24日周四 16:50写道：
